quiz.ipynb

```diff
@@ -0,0 +1,89 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "id": "eb4efd5c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['myself dimpal', '']\n",
+      "['i am live in bardoli', '']\n",
+      "['my favourite color is black', '']\n",
+      "['my hobby is travelling', '']\n",
+      "['i am studying at vidyabharti college trust', '']\n",
+      "['my age is 18', '']\n",
+      "[]\n"
+     ]
+    }
+   ],
+   "source": [
+    "import csv\n",
```

Comment:
Why you write import csv?

Reply:
The CSV import process saves time and prevents errors, by submitting data in a CSV (comma-separated value) file.

Reply:
its wrong. But CSV module is import so we can you object and function written this module in our program. like readline(), readlines() function and reader() and writer() object.